addOns/jwt/src/main/resources/org/zaproxy/zap/extension/jwt/resources/Messages.properties

```
@@ -0,0 +1,81 @@
+jwt.settings.title=JWT Settings
+jwt.toolmenu.settings=JWT Settings...
+ascanrules.jwt.name=JWT Scanner 
```

This isn't part of the `ascanrules` package, I'd suggest:
```suggestion
jwt.scanner.name=JWT Scan Rule
```

WRT the key name, It's technically a scan rule not a scanner, but that's nitt-picky.

---

sure, will update.